tests/core/request.test.ts

```diff
@@ -115,6 +116,52 @@ describe('Request properties', () => {
 
       await fetch('/').expect(200, `subdomains: `)
     })
+    describe('internal function tests', () => {
```

Comment:
wait, how are they internal if if `getSubdomiains` is used by `req.subdomains`?

Reply:
Those functions are not directly expected to be used by the end user.

Reply:
they're not exposed in the app package but you still can use them from `@tinyhttp/req`